os_net_config/objects.py

```diff
@@ -2231,7 +2233,9 @@ def __repr__(self):
             provider_info = f" (provider: {type(self.provider_data).__name__})"
         return (
```

Comment:
f"type='{self.remove_type}', {provider_info}), "
      f"provider='{self.provider}', "
      f"is_removed='{self.is_removed}'"
           
missing the comma